api/tests/integration/scripts/update-users-with-shouldValidateTermsOfService-to-true_test.js

```diff
@@ -0,0 +1,51 @@
+const { expect, databaseBuilder } = require('../../test-helper');
+
+const {
+  updateShouldValidateTermsOfServiceWhenUsersHaveNotSamlIdAndNotAssociatedWithSchoolingRegistration
+} = require('../../../scripts/update-users-with-shouldValidateTermsOfService-to-true');
+
+describe('Integration | Scripts | update-users-with-shouldValidateTermsOfService-to-true.js', () => {
+
+  describe('#updateShouldValidateTermsOfServiceWhenUsersHaveNotSamlIdAndNotAssociatedWithSchoolingRegistration', () => {
+
+    it('should not update user who have a samlId', async () => {
+      // given
+      const userId = databaseBuilder.factory.buildUser({ samlId: 'SAMLID' }).id;
+      await databaseBuilder.commit();
+
+      // when
+      const updatedUserIds = await updateShouldValidateTermsOfServiceWhenUsersHaveNotSamlIdAndNotAssociatedWithSchoolingRegistration();
+
+      // then
+      expect(updatedUserIds).to.not.include(userId);
+    });
+
+    it('should not update user who are associated with a schoolingRegistration', async () => {
+      // given
+      const userId = databaseBuilder.factory.buildUser().id;
+      const organizationId = databaseBuilder.factory.buildOrganization().id;
+      databaseBuilder.factory.buildSchoolingRegistration({ userId, organizationId });
+
+      await databaseBuilder.commit();
+
+      // when
+      const updatedUserIds = await updateShouldValidateTermsOfServiceWhenUsersHaveNotSamlIdAndNotAssociatedWithSchoolingRegistration();
+
+      // then
+      expect(updatedUserIds).to.not.include(userId);
+    });
+
```

Comment:
Rajouter un test `it should not update users who must not validate the terms of service`

Reply:
Voir https://github.com/1024pix/pix/pull/1332#discussion_r417332152
Il n'est pas possible d'implémenter cette fonctionnalité, donc l'ajout de test est caduc